### PR TITLE
Add steps to reboot computer and update tlgmr.

### PR DESCRIPTION
**Pull request by SuperMaZingCoder**
## Motivation
There are two additional required steps in the macOS installation instructions: rebooting and updating `tlgmr`.

## Overview / Explanation for Changes
Added these steps to documentation.

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by SuperMaZingCoder** [comment]
When installing `tlgmr` on macOS, the user must first reboot their machine and then update `tlgmr` before the other steps.

**Comment by behackl** [comment]
> When installing `tlgmr` on macOS, the user must first reboot their machine and then update `tlgmr` before the other steps.

I'm a bit curious, it's rare that rebooting is actually required; most of the time reloading the shell should be enough. Do you have a reference or so where this is mentioned?

**Comment by SuperMaZingCoder** [comment]
> I'm a bit curious, it's rare that rebooting is actually required; most of the time reloading the shell should be enough. Do you have a reference or so where this is mentioned?

I don't have somewhere this is mentioned, but I was just going through the installation steps and even after reloading the shell, it wasn't recognized, I'm not entirely sure why, though. I do know that I was unable to install the packages afterward because of an outdated `tlgmr` version.

File: docs/source/installation/mac.rst
Comment (WampyCakes) [inline docs/source/installation/mac.rst:52]:
```suggestion
Finally, run
```

File: docs/source/installation/mac.rst
Comment (WampyCakes) [inline docs/source/installation/mac.rst:45]:
```suggestion
Reboot your computer then ensure that the TeXLiveManager tool ``tlmgr`` works by running ``tlmgr
```

File: docs/source/installation/mac.rst
Comment (WampyCakes) [inline docs/source/installation/mac.rst:46]:
```suggestion
--version`` in the terminal. Now update ``tlmgr``
```

**Comment by SuperMaZingCoder** [comment]
Thanks for the changes! @WampyCakes

**Comment by WampyCakes** [comment]
I can't speak to whether a restart is actually required, but I imagine that it would be better to phrase this as a "restarting _may_ be necessary" kind of thing since @behackl seems skeptical that this is normal.

File: docs/source/installation/mac.rst
Comment (behackl) [inline docs/source/installation/mac.rst:46]: Indeed, I'd be a bit more comfortable with a weaker formulation:

```suggestion
Ensure that the TeXLiveManager tool ``tlmgr`` works by running ``tlmgr
--version`` in the terminal (a reboot might be necessary). Now update ``tlmgr``
```

(But if you disagree, I won't block this PR over it.)

**Review by behackl — APPROVED** [review]
LGTM, thank you for your contribution! I've added one additional comment and am awaiting your response before merging this.

File: docs/source/installation/mac.rst
Comment (WampyCakes) [reply docs/source/installation/mac.rst:46]: Agreed

**Comment by SuperMaZingCoder** [comment]
@behackl Sorry for the late reply! I made the changes and it should be good to merge now.

**Comment by behackl** [comment]
> @behackl Sorry for the late reply! I made the changes and it should be good to merge now.

No worries, all good -- thank you for your contribution! :-)

**Pull request merged**
